walletconnectv2/src/main/kotlin/com/walletconnect/walletconnectv2/client/WalletConnectClient.kt

```diff
@@ -85,6 +86,14 @@ object WalletConnectClient {
         }
     }
 
+    fun openConnection(onError: (String) -> Unit) {
+        relay.connect { errorMessage -> onError(errorMessage) }
+    }
+
+    fun closeConnection(onError: (String) -> Unit) {
+        relay.disconnect { errorMessage -> onError(errorMessage) }
+    }
+
```

Comment:
I would avoid naming it as `openConenction` and `closeConnection` it might be confusing with method such as `connect` and `disconnect`

Reply:
Maybe we should go this path?

```suggestion
object WebSockets {
    fun open(onError: (String) -> Unit) {
        relay.connect { errorMessage -> onError(errorMessage) }
    }

    fun close(onError: (String) -> Unit) {
        relay.disconnect { errorMessage -> onError(errorMessage) }
    }
}
```